src/App.jsx

```diff
@@ -1,11 +1,32 @@
+//! Imports
+import { Spinner } from 'react-bootstrap'
```

Comment:
can you use your own custom spinner? we don't prefer to use libraries directly. it is always difficult to change later on. you can export Spinner from your own component without doing anything right now. we are gonna use your one, and later on, when we want to change the spinner, it will be changed from one single method

```suggestion
import { MySpinner } from './components'
```